README.md

```diff
@@ -96,10 +96,14 @@ resource "instaclustr_cluster" "example" {
 
 ## Configuration
 ### Resources
-### Resource:  `instaclustr_cluster`  
-A resource for managing clusters on Instaclustr Managed Platform. A cluster contains a base application and several add-ons.
+### Resource:  `instaclustr_cluster` and Data Source `instaclustr_cluster_credentials`
```

Comment:
Could the data source be documented under a separate heading just below `instaclustr_cluster`?

Reply:
I have updated to document the new data source under a separate heading below `instaclustr_cluster`